### PR TITLE
Create outline for Kansas City file

### LINKED ISSUE #28: USERNAME githubteacher Track progress on GitHub Work Flow
# Briana's Workflow 🎉
- [x] create a branch 
- [x] add commits on my own branch
- [x] open a pull request
- [x] collaborate and have a conversation
- [x] make some more commits 
- [x] merge our branch into master

### PR DESCRIPTION
**Pull request by githubteacher**
This pull request adds a folder and file for Kansas City.

@halfaro1 @beardofedu @crichID @hollenberry Do you have a favorite type of food that I should add? 🍔 🎉 

This pull request resolves #28 ✨

### CODE REVIEW

File: githubteacher/USERNAME-githubteacher-kansascity.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+# Kansas City
+
+### Restaurants
```
Comment (MrTimms) [inline githubteacher/USERNAME-githubteacher-kansascity.md:3]: @githubteacher  Where is a good Thai place in KC?

Comment (VarunVats9) [inline githubteacher/USERNAME-githubteacher-kansascity.md:3]: @githubteacher Where can we get Indian Food in KC?

Comment (dany-phantom) [inline githubteacher/USERNAME-githubteacher-kansascity.md:3]: @githubteacher dog's hungry after skydiving yesterday
2572294_f520

**Comment by moisesespinosa** [comment]
Kansas City or St Louis ?

**Comment by Natalie8591** [comment]
$ git clone https://github.com/githubteacher/developers-jun-2016-americas.git
Cloning into 'developers-jun-2016-americas'...
fatal: unable to access 'https://github.com/githubteacher/developers-jun-2016-americas.git/': **Unknown SSL protocol error
 in connection to github.com:443**

**Pull request merged**